### PR TITLE
Use WixQuietExec to run command sliently

### PR DESCRIPTION
**Pull request by wli3**
Also add telemtery message to the installer

I am totally a designer -- 
The UI markup is really hard to use, and we are using WixStandardBootstrapperApplication, it is hard to add a custom page. Do you think I can append it blow the license for the first page?

message

The total install with actual timing, no cut
install_experience

### CODE REVIEW

**Comment by livarcocc** [comment]
@MattGertz for approval.

**Comment by livarcocc** [comment]
I think we should go ahead and merge it as this is a much better experience than a cmd window popping up during install. Also, how did you figure out how to do this all of a sudden? That's great.

However, I do think that progress window needs some love. cc @richlander for the text in case he wants to change anything.

As for the layout of the text, I think we need to add headers for the sections. Some spacing between them and we need to make the URLs actual links. But that can be a separate PR.

File: packaging/windows/clisdk/bundle.wxl [hunk 1]
```diff
@@ -51,4 +51,11 @@ Ready? Set? Let's go!</String>
   <String Id="FilesInUseDontCloseRadioButton">&amp;Do not close applications. A reboot will be required.</String>
   <String Id="FilesInUseOkButton">&amp;OK</String>
   <String Id="FilesInUseCancelButton">&amp;Cancel</String>
+  <String Id="FirstTimeWelcomeMessage">Welcome to .NET Core!
```
Comment (livarcocc) [inline packaging/windows/clisdk/bundle.wxl]: We need to improve how this text is displayed by adding headers, line breaks and making URLs links. But we can do that in a separate PR.

**Comment by livarcocc** [comment]
@dotnet-bot Test Debian8.2 x64 Debug Build

**Comment by wli3** [comment]
@dotnet-bot Test Debian8.2 x64 Debug Build

**Comment by wli3** [comment]
> Also, how did you figure out how to do this all of a sudden? That's great.

After trying out a blog post in the second page of search result
http://blogs.msmvps.com/rfennell/2013/02/25/running-an-external-command-line-tool-as-part-of-a-wix-install/

> However, I do think that progress window needs some love

Agree, however the "template we are using" -- [WixStandardBootstrapperApplication](http://wixtoolset.org/documentation/manual/v3/xsd/bal/wixstandardbootstrapperapplication.html), is very limited. We can only use one license page. No extra page. And only few basic UI elements are available. If we get out of the "Standard", we might need to write a CPP app ourself.

> As for the layout of the text, I think we need to add headers for the sections. Some spacing between them and we need to make the URLs actual links. But that can be a separate PR.

One side is I cannot find a good UI designer tool, I am using Paint to figure out the pixels position. Another side is this "TEXT" box can only put plain text, we can add some space and enter to make it better. Another route, I need more research to use _Richedit_. The memory of good old VB 6 days...

**Comment by wli3** [comment]
@dotnet-bot Test

**Comment by nguerrera** [comment]
```
10:14:03 17:13:59.866     2>CROSSGEN : error : Could not load file or assembly 'System.Security, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified. [/mnt/resource/j/workspace/dotnet_cli/release_2.0.0/debug_debian8.2_x64_prtest/src/redist/redist.csproj]
```

@livarcocc I'm seeing this locally and on my PR in release/2.0.0 branch. Thoughts on what caused this and how it got past CI?

**Comment by livarcocc** [comment]
@dotnet-bot Test this please.

@nguerrera It was a change that happened when stage0 got updated to the new SDK. This is caused by the implicit PTF for net461. I disabled that in the CLI. CI should be turning green again.

**Comment by nguerrera** [comment]
@livarcocc Yep, saw the trail of email right after I sent this. Thanks for fixing it.

**Comment by livarcocc** [comment]
@dotnet-bot Test Ubuntu x64 Release Build
